website/config/deprecation-workflow.js

```diff
@@ -11,6 +11,15 @@ self.deprecationWorkflow.config = {
     { handler: "silence", matchId: "remove-owner-inject" },
     { handler: "silence", matchId: "ember-modifier.function-based-options" },
     { handler: "throw", matchId: "deprecate-auto-location" },
-    { handler: "silence", matchId: "ember-string.add-package" }
+    { handler: "silence", matchId: "ember-string.add-package" },
+    { handler: "throw", matchId: "hds.dropdown.list-item.interactive" },
```

Comment:
this change revealed the dropdown id doesn't follow convention (missing `.components`), but I might address this in a separate PR